bot/exts/moderation/clean.py

```diff
@@ -615,6 +619,13 @@ async def clean_cancel(self, ctx: Context) -> None:
         await self._send_expiring_message(ctx, message)
         await self._delete_invocation(ctx)
 
+    @command()
+    async def purge(self, ctx: Context, users: Greedy[User], age: Optional[Union[Age, ISODateTime]] = None) -> None:
+        """Clean messages of users from all public channels up to a certain message age (10 minutes by default)."""
+        if age is None:
+            age = await Age().convert(ctx, "10M")
```

Comment:
I suppose discord.py does not apply decorators to default values? Ideally it would use an actual default, because I believe that gets shown in the help info for the command. Can that still be used somehow?

Reply:
With age being an offset from the time the command was ran, I'm not sure how we could get this to work with a deco evaluated at startup.

Reply:
Yeah I don't see a good way to do that. I used the same strategy for the default mute duration.